Update/kage_004.txt

```diff
@@ -5017,13 +5017,13 @@ void main()
 //rでも、それは私たちに対する最大の裏切りだと、なんで気付いてくれない？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "でも、それは私たちに対する最大の裏切りだと、なんで気付いてくれない？",
-			NULL, "And perhaps the most painful betrayal: Why doesn't she realize how we feel about this?", Line_Normal);
+			NULL, "But why doesn't she realize that's the most painful betrayal towards us?", Line_Normal);
 	ClearMessage();
 
 //r信頼しあって、心を通じ合って、…打算なんて考えない関係だからこそ、私たちは友達じゃなかったのか？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "信頼しあって、心を通じ合って、…打算なんて考えない関係だからこそ、私たちは友達じゃなかったのか？",
-			NULL, "We trusted each other. We had a deep understanding... So why does she have such a one-sided view of our friendship?", Line_Normal);
+			NULL, "Weren't we friends... because we trusted each other, had a deep understanding, and weren't trying to use each other for our own benefits?", Line_Normal);
```

Comment:
```suggestion
			NULL, "Weren't we friends... because we trusted each other, had a deep understanding, and weren't trying to use each other for our own benefit?", Line_Normal);
```